portal-api/api-tests/v1/gef/portal-activities.controller.api-test.js

```diff
@@ -291,4 +291,23 @@ describe('getUserInfo()', () => {
 
     expect(returnedUser).toEqual(expectedUserObject);
   });
+
+  it('Should return empty `firstname` and `lastname` when user does not exists', async () => {
+    // Wipe existing collections
+    await wipeDB.wipe([collectionName]);
+    await wipeDB.wipe([applicationCollectionName]);
+
+    // Fetch user from the collection
+    const userCollection = await db.getCollection('users');
+    const checker = await userCollection.findOne({ _id: '0' });
```

Comment:
we need to avoid mongo queries in tests. In the long run, I'd like to remove it altogether and keep only API calls to various endpoints.

can we use an API call instead to get these details?